bitnami/jenkins/templates/_helpers.tpl

```diff
@@ -27,3 +27,23 @@ Return the proper Docker Image Registry Secret Names
 {{- define "jenkins.imagePullSecrets" -}}
 {{- include "common.images.pullSecrets" (dict "images" (list .Values.image .Values.metrics.image .Values.volumePermissions.image) "global" .Values.global) -}}
 {{- end -}}
+
+{{/*
+Gets the host to be used for this application.
+When using Ingress, it will be set to the Ingress hostname.
+*/}}
+{{- define "jenkins.host" -}}
+{{- if .Values.ingress.enabled }}
+{{- .Values.ingress.hostname | default "" -}}
+{{- else -}}
+{{- index .Values (printf "%sHost" .Chart.Name) | default "" -}}
```

Comment:
Why setting it like this? Seems a bit hacky and complex taking into account that the chart name is not going to change

Reply:
Inherited from Magento, but I will simplify it